.github/workflows/release_info.sh

```diff
@@ -28,8 +32,10 @@ SEMVER_REGEX="\
 (\\+${FIELD}(\\.${FIELD})*)?$"
 
 # Used in downstream steps to determine if the release should be marked as a "prerelease" and if the build should build candidate release artifacts
-export IS_CANDIDATE=`[[ $NEW_TAG =~ $SEMVER_REGEX && ! -z ${BASH_REMATCH[4]} ]] && echo "true" || echo "false"`
```

Comment:
```
$ shellcheck .github/workflows/release_info.sh

In .github/workflows/release_info.sh line 35:
IS_CANDIDATE=$([[ $NEW_TAG =~ $SEMVER_REGEX && ! -z ${BASH_REMATCH[4]} ]] && echo "true" || echo "false")
                                               ^-- SC2236 (style): Use -n instead of ! -z.
```